### PR TITLE
Avatar badge

### PR DESCRIPTION
**Pull request by Sudhanshusoni45**

### CODE REVIEW

**Review by vp5h — APPROVED** [review]
looks good to me, Not sure but you can try using rems instead of px

**Review by NagaSatyaGeeth — APPROVED** [review]
code looks great !

File: Badge/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="badge-container">
+      <img src="/Badge/shopping-cart (1).png" alt="" />
```
Comment (VishalPatil18) [inline Badge/index.html]: **FIX:** Give value to the "alt" attribute as that helps for accessibility.

File: Badge/style.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  color: white;
+  font-weight: 900;
+  height: 18px;
+  width: 18px;
+  border: none;
+  background-color: var(--danger-color);
+  border-radius: 50%;
+
+  height: 20px;
+  width: 20px;
```
Comment (VishalPatil18) [inline Badge/style.css]: **Fix:** Height and width are mentioned twice here on lines 20-21 and 26-27. Remove the one which is not required.

**Review by VishalPatil18 — CHANGES_REQUESTED** [review]
Apart from a couple of fixes, rest everything looks good. Well Done Sudhanshu 👍

File: Badge/index.html (hunk 1 above)
Comment (Sudhanshusoni45) [reply Badge/index.html]: Sure vishal thanks for pointing these out i will fix these and commit

**Review by VishalPatil18 — APPROVED** [review]
Great, the code looks good now! Well done Sudhanshu 👍

**Pull request merged**